drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c

```diff
@@ -1540,6 +1544,14 @@ static int vidioc_g_selection(struct file *file, void *priv,
 			s->r.width = q_data->crop_width;
 			s->r.height = q_data->crop_height;
 			break;
+		case V4L2_SEL_TGT_CROP_BOUNDS:
```

Comment:
This change seems unrelated.

Reply:
Yes and no. I'll make it a second patch.

The only route for userspace to query the pixel aspect ratio is via ```VIDIOC_CROPCAP```, which goes through [v4l2_cropcap](https://elixir.bootlin.com/linux/latest/source/drivers/media/v4l2-core/v4l2-ioctl.c#L2451).
That calls our vidioc_g_pixelaspect function, but then follows it up with calls to g_selection for other properties, including these.
Where and how they are relevant for a video decoder is another questsion, but the call fails otherwise.

Oddly the only other decoder I can find which implements vidioc_g_pixelaspect is staging/media/meson/vdec, but it doesn't implement g_selection at all, so I don't believe there is any driver in mainline that actually implements this, but FFmpeg tries to read it.